modules/govuk_rabbitmq/manifests/remove_bindings.pp

```diff
@@ -0,0 +1,37 @@
+# == Define: govuk_rabbitmq::remove_bindings
+#
+# Removes the named rabbitmq bindings on a particular queue.
+# Multiple bindings can be removed at once by passing an array.
+#
+# We do not attempt to detect the condition of attempting to bind and
+# delete the same routing_key to a queue in a single puppet run,
+# so for clarity this is best used adjacent to any other config for the same queue.
+#
+# === Parameters
+#
+# [*amqp_vhost*]
+#   The RabbitMQ vhost from which to delete bindings.
+#
+# [*amqp_exchange*]
+#   The RabbitMQ exchange from which to delete bindings.
+#
+# [*amqp_queue*]
+#   The RabbitMQ queue from which to delete bindings.
+#
+define govuk_rabbitmq::remove_bindings (
```

Comment:
Actually, it can handle an array of keys - that's what the `$routing_key = $title` achieves.

Many thanks for all the comments btw :)